addons/website_slides/models/slide_channel.py

```diff
@@ -52,17 +54,22 @@ def _recompute_completion(self):
              ('completed', '=', True),
              ('slide_id.is_published', '=', True),
              ('slide_id.active', '=', True)],
-            ['channel_id', 'partner_id'],
+            ['channel_id', 'partner_id', 'slide_id:array_agg'],
```

Comment:
if `next_slide_id` is computed non-stored, it should be `slide_id:count_distinct`